main.go

```diff
@@ -99,10 +99,11 @@ func createStep(logger log.Logger, logFormatter string) (step.XcodeTestRunner, e
 	utils := step.NewUtils(logger)
 
 	xcodeCommandRunner := xcodecommand.Runner(nil)
+	rawXcodeRunner := xcodecommand.NewRawCommandRunner(logger, commandFactory)
```

Comment:
`rawXcodeRunner` is only used if `logFormatter == step.XcodebuildTool`. I suggest creating the instance in the switch - case block.